ui/lib/kv/addon/components/kv-data-fields.hbs

```diff
@@ -1,5 +1,5 @@
 {{#let (find-by "name" "path" @secret.allFields) as |attr|}}
-  {{#if @previousVersion}}
+  {{#if @isEdit}}
```

Comment:
Simplified this to a boolean since it's already being used as one, and that way we can be explicit about when it's edit because sometimes the user cannot read the previous version